components/squad/squad_row.lua

```diff
@@ -108,8 +126,10 @@ function SquadRow:newteam(args)
 
 		local newTeam = args.newteam:lower()
```

Comment:
Throws exception is newteam is not set

Reply:
`args.newteam` is checked on line 120

Reply:
My bad, must have missed that while going through